openlibrary/macros/LoanStatus.html

```diff
@@ -112,13 +113,13 @@
   </div>
 
 $if ocaid and secondary_action and availability.get('is_printdisabled'):
-  $:macros.BookPreview(ocaid)
+  $:macros.BookPreview(ocaid, linkback=not no_index)
   $:macros.BookSearchInside(ocaid)
 
 $:post
 
 $if ocaid and daisy:
-  $:macros.daisy('/ia/%s/daisy' % ocaid, protected=availability.get('is_printdisabled'))
+  $:macros.daisy('%s/daisy' % doc.url(), protected=availability.get('is_printdisabled'))
```

Comment:
@mekarpeles 

was this a typo? it looks like this is what's breaking author pages at the moment.

```suggestion
  $:macros.daisy('%s/daisy' % doc.url, protected=availability.get('is_printdisabled'))
```

Reply:
Great catch @shanamatthews ! Yep; looks like it's also breaking search pages :P

Reply:
We may have do dynamically compose this url in a more sophisticated way. Removing the `()` will break the contract for editions and works. Unfortunately, this "method" presents itself inconsistently across Open Library things, so the approach of using `doc.url` / `doc.url()` doesn't seem ideal for what we're trying to achieve. Presumably one of these is a property and the  other an instance method, so there also isn't a great way to bring them into alignment without more work than it's probably worth. Instead, we should create an endpoint, similar to `https://openlibrary.org/ia/borrow/{ocaid}?type=daisy` which resolves to the daisy on archive.org and sidestep the unneeded complexity.